view/layout.html

```diff
@@ -10,411 +10,83 @@
-                  </ul>
-                </li>
-                <!--
-                <li><a href="{{appUrl}}/databases">Databases</a>
-                  <ul>
-                    <li><a href="{{appUrl}}/datastore">Built-in Datastore</a></li>
-                    <li><a href="{{appUrl}}/databases/dynamodb">Amazon DynamoDB</a></li>
-                    <li><a href="{{appUrl}}/databases/couchdb">CouchDB</a></li>
-                    <li><a href="{{appUrl}}/databases/mongodb">MongoDB</a></li>
-                    <li><a href="{{appUrl}}/databases/redis">Redis</a></li>
-                    <li><a href="{{appUrl}}/databases#additional">More...</a></li>
-                  </ul>
-                </li>
-                <li><a href="{{appUrl}}/files">File Storage</a>
-                  <ul>
-                    <li><a href="{{appUrl}}/files#amazon">Amazon s3</a></li>
-                    <li><a href="{{appUrl}}/files#google">Google Cloud</a></li>
-                    <li><a href="{{appUrl}}/files#microsoft">Mirosoft Azure</a></li>
-                    <li><a href="{{appUrl}}/files#rackspace">Rackspace Cloud Files</a></li>
-                    <li><a href="{{appUrl}}/files#sftp-ssh">SSH / SFTP</a></li>
-                  </ul>             
-                </li>
-                -->
-                <li><a href="{{appUrl}}/keys">Role Based Access Control</a>
-                  <ul>
-                    <li><a href="{{appUrl}}/keys">API Access Keys</a></li>
-                    <li><a href="{{appUrl}}/roles">System Roles</a></li>
-                    <li><a href="{{appUrl}}/roles#custom-role-checks">Custom Roles</a></li>
-                  </ul>
-                </li>
-                <li><a href="{{appUrl}}/api">Developer API</a>
-                  <ul>
-                    <li><a href="{{appUrl}}/api">Getting Started</a></li>
-                    <li><a href="{{appUrl}}/api#keys">API Access Keys</a></li>
-                    <li><a href="{{appUrl}}/api#rest">HTTP REST API</a></li>
-                    <li><a href="{{appUrl}}/sdk">Client SDK</a></li>
-                    <li><a href="{{appUrl}}/gateways">Hot Code Gateways</a></li>
-                  </ul>
-                </li>
-                <li><a href="{{appUrl}}/contact">Contact Us</a></li>
-              </ul>
-            </nav>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-  <!--end mobile menu  area -->   
-
+    <nav class="topNav" role="navigation">
+      <a class="topNav-logo" href="{{appUrl}}" >
+        <img src="{{appUrl}}/img/logo.png" alt="hook.io">
+      </a>
+      <ul class="topNav-list">
+        <li class="topNav-listItem">
+          <a class="topNav-link" href="{{appUrl}}/pricing">pricing</a>
+        </li>
+        <li class="topNav-listItem">
+          <a class="topNav-link" href="{{appUrl}}/login">log in</a>
+        </li>
+      </ul>
+    </nav>
+    <input type="checkbox" class="mobileNavCheckbox" id="hamburger">
```

Comment:
@Marak `.mobileNav`, `.mobileNavButton`, and `.mobileNavCheckbox` should only be visible when logged in